.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules/
```

Comment:
mind keeping node_modules and *.log out of the .gitignore? I prefer all platform-related ignores to be set globally rather than in each project.